AutocompleteClient/FW/Logic/Request/Builder/CIOAnswerQuizQueryBuilder.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  CIOAnswerQuizQueryBuilder.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+/**
+ Request builder for creating an browse query.
+ */
+public class CIOAnswerQuizQueryBuilder {
+
+    /**
+     The id of the quiz
+     */
+    let quizId: String
+
+    /**
+     A list of answers in the format ?a=[option_id_1]&a=[option_id_2] with one a parameter for each question. The expected format for each parameter depends on the question type, as described below
+     */
+    var answers: [String]?
+    
+    /**
+     Creata an Answer Quiz query builder
+     
+     - Parameters:
+        - quizId: The id of the quiz
+        - answers: A list of answers in the format ?a=[option_id_1]&a=[option_id_2] with one a parameter for    each question. The expected format for each parameter depends on the question type, as described below
+     */
+    
+    public init(quizId: String, answers: [String]) {
```

Comment:
Since `quizId` is the only required parameter, I think we can remove `answers` as a parameter from the `init` function.